XenModel/XenAPI/FriendlyErrorNames.resx

```diff
@@ -1478,12 +1484,12 @@ Authorized Roles: {1}</value>
   <data name="SR_BACKEND_FAILURE_99" xml:space="preserve">
     <value>Only 1 LUN may be used with shared LVM</value>
   </data>
-  <data name="SR_DETACHED_ON_MASTER" xml:space="preserve">
-    <value>The SR is currently detached on the master.</value>
-  </data>
   <data name="SR_DEVICE_IN_USE" xml:space="preserve">
     <value>The SR operation cannot be performed because a device underlying the SR is in use by the server.</value>
   </data>
+  <data name="SR_DOES_NOT_SUPPORT_MIGRATION" xml:space="preserve">
+    <value>You attempted to migrate a VDI on SR which doesn't have snapshot capability</value>

```

Comment:
The message is used for both _onto_ and _from_ and it refers to the "vdi_snapshot" sm capability.
We decided to change it to "You attempted to migrate a VDI to or from an SR that doesn't support migration"